### DISCUSSION TITLE
GitHub PRO badges is a joke?

### DISCUSSION THREAD

**Discussion by WhoIM01** [community/community#37019] [Other Feature Feedback, Questions, & Ideas]
Github show up today with new "PRO" banners bellow pictures in users profiles and also in miniatures after mouseover . Almost 80% of people i searched for have this banner. I just want to understand what kind of utility does new badge bring off to the community when there is only a "PRO" and no badge.

**Comment by Cauatn** [comment]
The GitHub Pro subscription offers several advantages, including access to private repositories, unlimited collaborators for private projects, prioritized support, GitHub Actions and Packages usage, GitHub Pages hosting, Code Owners feature, and advanced security options like Dependabot. These benefits cater to individuals and organizations seeking enhanced collaboration, privacy, and support on the GitHub platform.
I hope this helps your question.
